### DISCUSSION TITLE
What should I do before install NvChad if I have already installed and configured NeoVim?

### DISCUSSION THREAD

**Discussion by griiid** [NvChad/NvChad#1665] [Q&A] [answered]
Hello,

I've searched keyword "installed" in both Issues & Discussions, and I didn't find any discussion which could answer my question.

I already installed NeoVim, and have installed lot of plugins, written a lot of configurations.
Now I want to try NvChad, but I don't know what should I do before installing it.
I guess I should uninstall NeoVim, and then delete `~/.config/nvim/` & `~/.vimrc` & related `.vim` files.
I'm not sure what more should I do, can someone list a to-do list for me before install NvChad?

Thanks.

**Comment by siduck** [comment, marked as answer]
just make a backup of your config and then migrate your config to nvchad, check nvchad.com for nvchad's custom config syntax

**Reply by griiid** [reply]
> just make a backup of your config and then migrate your config to nvchad, check nvchad.com for nvchad's custom config syntax

Should I:

1. execute PlugUninstall?
2. remove `~/.config/nvim/` & `~/.vimrc` & related `.vim` files after make a backup?
3. uninstall old NeoVim?

**Reply by siduck** [reply]
first backup your config then uninstall old nvim

```bash
rm -rf ~/.config/nvim
rm -rf ~/.local/share/nvim
rm -rf ~/.cache/nvim
rm -rf ~/.vimrc
```
